packages/astro/test/sourcemap.test.js

```diff
@@ -14,9 +14,4 @@ describe('Sourcemap', async () => {
 		const counterMap = dir.find((file) => file.match(/^Counter\.\w+\.js\.map$/));
 		expect(counterMap).to.be.ok;
 	});
-
-	it('Builds non-empty sourcemap', async () => {
```

Comment:
@bluwy I noticed you were the last person to change this file. Could you please confirm that this is no longer true? Now the `entry.mjs` imports `renderers.mjs`, which means we should have `sources: []`?

Reply:
Yeah it looks like with the new chunks, the sources are moved to `renderers.mjs.map` instead. Maybe we can update the test to check that file instead?

Reply:
Great! I will do that :) thank you!